src/main/java/no/fdk/model/fuseki/action/CompactAction.java

```diff
@@ -16,18 +20,27 @@
 import java.util.stream.Stream;
 
 @Slf4j
+@Component
 public class CompactAction extends BaseActionREST {
     @Override
     protected void doGet(HttpAction action) {
-        log.info("Started compaction process");
+        doCompact(action.getDataset());
+    }
 
-        DatasetGraph datasetGraph = action.getDataset();
+    private void doCompact(DatasetGraph datasetGraph) {
+        String datasetName = datasetGraph.getContext().getAsString(Symbol.create("name"));
+        log.info("Started compaction process for dataset '%s'".formatted(datasetName));
 
         DatabaseMgr.compact(datasetGraph);
 
         removeUnusedFiles(datasetGraph);
 
-        log.info("Finished compaction process");
+        log.info("Finished compaction process for dataset '%s'".formatted(datasetName));
+    }
+
+    public void doCompactForDatasetPath(String datasetPath) {
```

Comment:
Holder med at metode signatur er annerledes

```suggestion
    public void compact(String datasetPath) {
```